packages/components/messages/src/error-message/error-message.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import ErrorMessage from './error-message';
+
+const meta: Meta<typeof ErrorMessage> = {
+  title: 'components/Messages/ErrorMessage',
+  component: ErrorMessage,
```

Comment:
Like in `WarningMessage`, setting an `intlMessage` breaks the example component.
```suggestion
  component: ErrorMessage,
    argTypes: {
    intlMessage: { control: { disable: true } },
  },
```